File: Java Code/Basics/Mensuration/Triangle.java
import java.util.*;

public class Triangle{
  public static void main(String[] args){
    Scanner sc=new Scanner(System.in);
    System.out.print("Enter first side : ");
    float a=sc.nextFloat();
    System.out.print("Enter second side : ");
    float b=sc.nextFloat();
    System.out.print("Enter third side : ");
    float c=sc.nextFloat();
    float S=(a+b+c)/2;
    float A=(float)(Math.sqrt(S*(S-a)*(S-b)*(S-c)));
    System.out.println("Perimeter : "+2*S);
    System.out.println("Area = "+A);
  }
}

File: Javascript Code/Mensuration/Triangle.js
let a=parseInt(prompt("Enter the first side of triangle : "));
let b=parseInt(prompt("Enter the second side of triangle : "));
let c=parseInt(prompt("Enter the third side of triangle : "));
let s=(a+b+c)/2;
console.log(`Area : ${Math.sqrt(s*(s-a)*(s-b)*(s-c)).toFixed(2)} sq unit.`);
console.log(`Perimeter : ${a+b+c} unit.`);